gym/envs/toy_text/blackjack.py

```diff
@@ -81,6 +84,8 @@ def __init__(self, natural=False):
         # Flag to payout 1.5 on a "natural" blackjack win, like casino rules
         # Ref: http://www.bicyclecards.com/how-to-play/blackjack/
         self.natural = natural
+        #Flag for allowing doubling down
```

Comment:
this comment should rather be in the docstring (could you also describe what natural stands for while at it then ^_^)

Reply:
Thanks, updated in new commit